Resprites most organs and augments, adds icons for new augments

SomeAngryMiner opened the pull request:
## Ah shit, here we go again.
Technically a continuation of #30227 's work , this PR aims to standarize the way the organ sets for most non-admin races are seen, and improves them somewhat, making them compliant to today's standards such as how outlines are handled in newer sprites, or animation frames and tools for older assets such as posibrain, wich instead of modifying the time a frame stayed, repeated the sprite over and over.

### The full list of organs changed is:
+ Brain
+ Brain ghost
+ Appendix
+ Inflamed appendix
+ Liver
+ Cortical stack (Now animated)
+ Vox heart
+ Vox Lung
+ Compound eyes
+ Lungs
+ Kidneys
+ Stomach
+ Eyes
+ Mechanical brain (Currently used as cyborg ghost brain)
+ Artificial heart
+ Artificial lungs
+ Artificial eyes
+ Internal battery (used in FBPs and IPCs, broken state also included)
+ Artificial Liver
+ Artificial Kidneys
+ Voicebox (Used by GAS)
+ Armblade implant
+ Cyberclaw implant
+ Subdermal armor
+ Toolset augment
+ Surgical augment (new sprite, old one used the old toolset augment sprite)
+ Integrated circuit augment
+ Security HUD lens (new sprite, old used booster sprite)
+ Medical HUD (Ditto above)
+ Booster
+ Skrell variants of human organs
+ Unathi variants of human organs
+ Ascent Organs
+ Ascent mechanical controller
+ Posibrain (and all required states)
+ Posibrain Shackles


## Comparation sheet, in the above order.
![image](https://user-images.githubusercontent.com/53237389/112542703-a30a4400-8d93-11eb-8dff-468fb2533334.png)

# Probably pending approval by species maintainers on alien organ colorations.

Im an idiot and i did all these changes in master FUCK

## Changelog
:cl: AzlanonPC
add: implantable HUDs and multitools have different sprites for each variation.
imgadd: Most organs, both organic and mechanical, have received a visual revision.
/:cl:

Commit:
Better circuit board examines

SierraKomodo committed:
Add spawn delay to exoplanet awakening

Spookerton committed:
various spawn to timer conversions

remove unused helper /datum/construction*

spawn -> timers for roundstart

resources are timer gated, not spawn

smoke effect coughedtime -> timers

capture map -> timers

no intercom spawns

blood and slime timers

slime AI timers

phoron combust timer

pointing timer

phlogiston hotspot timer

phoron airlock hotspot timer

singularity temp timer

door justzap timer

not sure why this machine exists but hey, timer

remove unused turf proc Sandbox_Spacemove

ai camera tracking uses timers

narnar and cascade portals use timers for pulling

tests

SierraKomodo committed:
Prevent unremoval items and natural weapons from embedding

SierraKomodo committed:
Fix train hitching

Ryan180602 committed:
no coughing in the classroom

Ryan180602 committed:
splishfix

SierraKomodo committed:
Tweaks deck 1 cryo maint pipe layout

Less confusing to look at on the map as pipes no longer run over eachother in turns and look like 4-way junctions.

SomeAngryMiner committed:
Merge pull request #1 from Baystation12/dev

Keeping up to date

SomeAngryMiner committed:
Organ visuals redesign

Vimas21 committed:
Lowered Groin Multiplier to 1.5

SierraKomodo reviewed (changes requested):
Code looks fine.

Changelog has a typo - `mechanica`

Otherwise waiting for lore review on the sprites.

SierraKomodo reviewed (commented):
Just waiting for lore now. Giving myself a checklist cause I'll forget otherwise:
- [x] IPC
- [x] Skrell
- [x] Vox
- [x] Ascent
- [x] GAS
- [x] Unathi

SierraKomodo committed:
Merge pull request #30418 from Ryan180602/nocough

FBPs no longer cough in smoke

SierraKomodo committed:
Merge pull request #30420 from Ryan180602/splishfix

Fixes oversight with pouring reagents into dispensers

Bay12Bot committed:
Automatic changelog generation for PR #30418 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #30420 [ci skip]

MistakeNot4892 commented:
Would you object to some of these being ganked for Nebula?

SomeAngryMiner commented:
Be my guest, as long as you link back to the original PR, here

SierraKomodo committed:
Update wall damage processing

SierraKomodo committed:
Update window damage processing

CrimsonShrike committed:
Makes augments jump gooder more reliably

PsiOmegaDelta committed:
Automatic changelog update

SierraKomodo committed:
Merge pull request #30423 from Vimas21/groin_reduction

Lowered Groin Multiplier to 1.5

Bay12Bot committed:
Automatic changelog generation for PR #30423 [ci skip]

PsiOmegaDelta committed:
Automatic changelog update

Spookerton committed:
Merge pull request #30305 from Ilysen/machine-circuits

construction skill lets you examine some machines and their circuit boards to learn more about them

Bay12Bot committed:
Automatic changelog generation for PR #30305 [ci skip]

Spookerton reviewed (changes requested):
Since we're in here and some changes have been asked for, please reset and rebase instead of merging dev into your feature branch.

SomeAngryMiner committed:
posibrain  sprite is posigone

juxjux9930 committed:
Makes fleet uniform choices more uniform

PsiOmegaDelta committed:
Automatic changelog update

CrimsonShrike committed:
Buffs light exosuit and spherical pod

SierraKomodo committed:
Merge pull request #30432 from CrimsonShrike/JumpLegsGood

Makes augments jump gooder more reliably

Spookerton committed:
Merge pull request #30352 from SierraKomodo/exo-awakening-delay

Add spawn delay to exoplanet awakening

Spookerton committed:
Merge pull request #30387 from SierraKomodo/fix-natural-weapon-embedding

Prevent unremoval items and natural weapons from embedding

Spookerton committed:
Merge pull request #30419 from SierraKomodo/fix-trolley-linking

Fix train hitching

Bay12Bot committed:
Automatic changelog generation for PR #30352 [ci skip]

Spookerton committed:
Merge pull request #30421 from SierraKomodo/deck1-pipe-less-confusing

Tweaks deck 1 cryo maint pipe layout

Bay12Bot committed:
Automatic changelog generation for PR #30387 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #30419 [ci skip]

CrimsonShrike committed:
Fixes unpowered exosuits not recharging

Turns out you should test stuff

SierraKomodo committed:
Merge pull request #30376 from Spookerton/spkrtn/cng/salmon-run-1

swaps out some spawns for timers

MuckerMayhem committed:
minor updates to exo-awakening

Fixed the z-level detection not working on players inside things like mechs, lockers, sleepers, etc. 

Drastically lowered the spawn chance of deadlier meat-mob variants, and added a generic spider type to the spiders mob list.

PsiOmegaDelta committed:
Automatic changelog update

Commit:
Pixel begone

should fix issue #30443 the rogue pixel is gone

Spookerton committed:
Merge pull request #30429 from SierraKomodo/window-wall-damage-update

Windows & Wall damage update

Bay12Bot committed:
Automatic changelog generation for PR #30429 [ci skip]

Spookerton committed:
Merge pull request #30444 from JAR-OSKI/Pixel-begone

Fixes Issue #30443

Bay12Bot committed:
Automatic changelog generation for PR #30444 [ci skip]

Spookerton committed:
Merge pull request #30440 from MuckerMayhem/exo-awake-updates

minor updates to exo-awakening

Bay12Bot committed:
Automatic changelog generation for PR #30440 [ci skip]

SierraKomodo committed:
Merge pull request #30434 from juxtaposed9930/uniformuniforms

Torch Fleet Uniformity

Bay12Bot committed:
Automatic changelog generation for PR #30434 [ci skip]

Imienny1 committed:
Torch and Skrell scoutship no longer have access to thermals

juxjux9930 committed:
fixes my hubris

Spookerton committed:
Merge pull request #30453 from juxtaposed9930/fixywixy

Fixes Fleet Uniform Lies

Bay12Bot committed:
Automatic changelog generation for PR #30453 [ci skip]

CrimsonShrike committed:
Fixes icon cache caching icon layers when it should not

SierraKomodo committed:
Merge pull request #30437 from CrimsonShrike/ExoSeats

Buffs light exosuit and spherical pod

SierraKomodo committed:
Merge pull request #30439 from CrimsonShrike/FixExoRecharge

Fixes unpowered exosuits not recharging

Bay12Bot committed:
Automatic changelog generation for PR #30437 [ci skip]

SierraKomodo committed:
Merge pull request #30454 from CrimsonShrike/ExosuitCacheCorrupted

Fixes icon cache caching icon layers when it should not

SierraKomodo committed:
Merge pull request #30450 from Imienny1/ThermalRemoval2

Torch and Skrell scoutship no longer have access to thermals

Bay12Bot committed:
Automatic changelog generation for PR #30450 [ci skip]

juxjux9930 committed:
Toolbelts can hold geiger counters

SierraKomodo committed:
Merge pull request #30463 from juxtaposed9930/geigerbelt

Toolbelts can hold geiger counters

Bay12Bot committed:
Automatic changelog generation for PR #30463 [ci skip]

Spookerton commented:
@SierraKomodo I'm calling the last two good. Still needs reset and rebase, though.

SomeAngryMiner commented:
Alright, time to reset and rebase.

SomeAngryMiner committed:
Merge branch 'dev' of https://github.com/SomeAngryMiner/Baystation12 into dev

SomeAngryMiner commented:
done-so, i'll open a new pull request to clean up the commit list, in a second.
